src/components/Search/Search.url.svg

```diff
@@ -0,0 +1,7 @@
+<svg version="1.1" id="Search" 
```

Comment:
Looks like a duplicate of `./Search.svg` file. Can we reuse the existing file?

Reply:
I guess we could it's just a renaming

Reply:
Done